shared/app.js

```diff
@@ -73,7 +74,9 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
I see. So can we kill that default fallback entirely and plug in default router via same mechanism as for the custom router?